src/w3c/templates/headers.js

```diff
@@ -3,11 +3,25 @@ import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
 
+function decorateSpecTitle(specTitleElem) {
+  specTitleElem.classList.add("title", "p-name");
+  specTitleElem.id = "title";
```

Comment:
given suggestion below, this line can go away, and we can drop `decorateSpecTitle()`, but keep `specTitleElem.classList.add("title", "p-name");` as it's the only thing we need to do to the `h1`.